### DISCUSSION TITLE
New to this and LOVING it

### DISCUSSION THREAD

**Discussion by jordanbr20** [lllyasviel/Fooocus#1513] [General]
So I am looking for some help, I recently found out about this and found out I love AI! I first found this because of the AI influencer youtube video but have found that I love this for so many other reasons. But I am having a hard time wrapping my head around all of this, their are sooo many settings, so many additions like models, etc. Where would be a good spot to learn about all of those?

Also, Lets say I get a face I want, and a Body I want, how can I keep that 100% consistent? For instance, I create a girl with freckles, do a face swap, it never turns out with the same freckles... but the face looks similar.

**Comment by mashb1t** [comment]
Great to hear that Fooocus has sparked your interest!
https://stable-diffusion-art.com/beginners-guide/ is a great way to start to get to know the most common terms and what's hiding behind them.

For the beginning, i'd in your place stick with image prompt. When you are more experienced, you can start training your own LoRAs with e.g. https://github.com/bmaltais/kohya_ss?tab=readme-ov-file#tutorials to get consistent results of a subject.

Hope that helped and happy to have you here :)
